src/Core/src/Handlers/Layout/LayoutHandler.Gtk.cs

```diff
@@ -124,9 +124,21 @@ public override Size GetDesiredSize(double widthConstraint, double heightConstra
 
 #endif
 
+		private void UpdateVisibility(Visibility visibility)
+		{
+			if (visibility == Visibility.Visible)
+				PlatformView.Show();
+			else
+				PlatformView.Hide();
+		}
+
 		[MissingMapper]
 		public void UpdateZIndex(IView view) => throw new NotImplementedException();
 
+		static void MapVisibility(ILayoutHandler handler, ILayout layout)
+		{
+			(handler as LayoutHandler)?.UpdateVisibility(layout.Visibility);
```

Comment:
@webwarrior-ws shouldn't this be a static cast?

Reply:
Like this: `((LayoutHandler)handler)` ?

Reply:
yes?

Reply:
Done